engine/org.eclipse.birt.report.engine.emitter.pdf/src/org/eclipse/birt/report/engine/emitter/pdf/PDFPageDevice.java

```diff
@@ -867,4 +873,33 @@ public void setDefaultFontPdfA(String defaultFont) {
 	public String getDefaultFontPdfA() {
 		return this.defaultFontPdfA;
 	}
+
+	/**
+	 * Set the including of a font CIDSet stream the document. When set to true, a
+	 * CIDSet stream will be included in the document. When set to false, no CIDSet
+	 * stream will be included.
+	 */
+	private void setPdfIncludeCidSet() {
+		if (this.userProperties != null && this.userProperties.containsKey(PDFPageDevice.PDF_FONT_CID_SET))
+			this.includeFontCidSet = Boolean
+					.parseBoolean(this.userProperties.get(PDFPageDevice.PDF_FONT_CID_SET).toString());
+	}
+
+	/**
+	 * Set the including of a font CIDSet stream the document
+	 *
+	 * @param includeFontCidSet include CIDSet stream of a font to the document
+	 */
+	public void setPdfIncludeCidSet(Boolean includeFontCidSet) {
+		this.includeFontCidSet = includeFontCidSet;
+	}
+
+	/**
+	 * Get the instruction to include CIDSet stream of fonts
+	 *
+	 * @return the CIDSet shall be included
+	 */
+	public boolean getPdfIncludeCidSet() {
```

Comment:
It's more common to use "is" as the prefix for a boolean "getter".   The "Pdf" in the name seems redundant.

Reply:
Done.